### PR TITLE
Revise README for UDF

### PR DESCRIPTION
**Pull request by scalarindetail**
The PR add contents to README regarding to UDF.

### CODE REVIEW

**Comment by feeblefakie** [comment]
@scalarindetail I updated the README. 
Can you check and add a StateUpdater.java function as an example ?

Ben, can you check the English please  ?

**Comment by scalarindetail** [comment]
@feeblefakie 
Thanks for the feedback.
I added a new class `com.scalar.client.tool.emulator.function.StateUpdater` and revised the related part in README.md.
Please check it again and let me know if the UDF is confusing to the readers since the behaviour of  `com.scalar.client.tool.emulator.function.StateUpdater` is very similar with `com.scalar.client.tool.emulator.contract.StateUpdater`.

Thanks.

**Comment by scalarindetail** [comment]
Ben will help revise the English expression.

**Comment by scalarindetail** [comment]
Ben revised the README and I have committed his advice.

File: emulator/README.md [hunk 1]
```diff
@@ -39,26 +37,60 @@ Exit the emulator with `exit` or by ctrl-d (EOF).
...
+For example, the `state-updater-contract` can be executed as follows.
 
 ```
-scalar> execute state-updater {"asset_id": "Y", "state": 1}
+scalar> execute state-updater-contract {"asset_id": "Y", "state": 1}
 ```
 
+## Register a user-defined function (UDF)
+
+UDF is a business logic to update the mutable database in the Scalar DL network. The
```
Comment (feeblefakie) [inline emulator/README.md]:
```suggestion
User-defined function (UDF) is a business logic to update the mutable database in the Scalar DL network. The
```

**Pull request merged**